ueman/feedback
# Add 36 languages translations

## :scroll: Description


* Add 36 languages translations
* Remove example/pubspec.lock

## :bulb: Motivation and Context



* Some users can't read English and need to support more languages.

* pubspec.lock is ignored by .gitignore. url will be rewritten if PUB_HOSTED_URL is modified. So remove this file.

## :green_heart: How did you test it?

* Change the device language and observe if the display is normal and if the text is too long.
* Delete pubspec.lock and recompile successfully, so deleting this file does not affect the compilation.

## :pencil: Checklist

- [ ] I added tests to verify changes


## :crystal_ball: Next steps

Except for Chinese, the translation cannot be guaranteed to be absolutely correct and needs to be further optimized by Native Speaker.
